SDVModTest/UIElements/LocationOfTownsfolk.cs

```diff
@@ -25,77 +26,86 @@ class LocationOfTownsfolk : IDisposable
+                        { "Blacksmith", new KeyValuePair<int, int>(852, 388) },
+                        { "JoshHouse", new KeyValuePair<int, int>(740, 320) },
+                        { "SandyHouse", new KeyValuePair<int, int>(40, 40) },
+                        { "Tunnel", new KeyValuePair<int, int>(0, 0) },
+                        { "CommunityCenter", new KeyValuePair<int, int>(692, 204) },
+                        { "Backwoods", new KeyValuePair<int, int>(460, 156) },
+                        { "ElliottHouse", new KeyValuePair<int, int>(826, 550) },
+                        { "SebastianRoom", new KeyValuePair<int, int>(732, 148) },
+                        { "BathHouse_Entry", new KeyValuePair<int, int>(576, 60) },
+                        { "Greenhouse", new KeyValuePair<int, int>(0, 0) },
+                        { "Sewer", new KeyValuePair<int, int>(380, 596) },
+                        { "WizardHouse", new KeyValuePair<int, int>(196, 352) },
+                        { "Trailer", new KeyValuePair<int, int>(780, 360) },
+                        { "Forest", new KeyValuePair<int, int>(80, 272) },
+                        { "Woods", new KeyValuePair<int, int>(100, 272) },
+                        { "WitchSwamp", new KeyValuePair<int, int>(0, 0) },
+                        { "ArchaeologyHouse", new KeyValuePair<int, int>(892, 416) },
+                        { "FishShop", new KeyValuePair<int, int>(844, 608) },
+                        { "Saloon", new KeyValuePair<int, int>(714, 354) },
+                        { "LeahHouse", new KeyValuePair<int, int>(452, 436) },
+                        { "Town", new KeyValuePair<int, int>(680, 360) },
+                        { "Mountain", new KeyValuePair<int, int>(762, 154) },
+                        { "BusStop", new KeyValuePair<int, int>(516, 224) },
+                        { "Railroad", new KeyValuePair<int, int>(644, 64) },
+                        { "SkullCave", new KeyValuePair<int, int>(0, 0) },
+                        { "BathHouse_WomensLocker", new KeyValuePair<int, int>(576, 60) },
+                        { "Beach", new KeyValuePair<int, int>(790, 550) },
+                        { "BathHouse_MensLocker", new KeyValuePair<int, int>(576, 60) },
+                        { "Mine", new KeyValuePair<int, int>(880, 100) },
+                        { "WitchHut", new KeyValuePair<int, int>(0, 0) },
+                        { "AnimalShop", new KeyValuePair<int, int>(420, 392) },
+                        { "SamHouse", new KeyValuePair<int, int>(612, 396) },
+                        { "WitchWarpCave", new KeyValuePair<int, int>(0, 0) },
+                        { "Club", new KeyValuePair<int, int>(60, 92) }
+                };
+
+        #endregion
+
+        public LocationOfTownsfolk(ModOptions modOptions, ModConfig config, IModHelper helper)
         {
             _helper = helper;
-            _options = options;
+            _townspeopleOptions = config.Townspeople;

```

Comment:
Moving this to modconfig means you won't be able to select different NPC's in different saves. This needs to be back where I had it originally.